Chat space画面4/18時点最新２

FUM01ichikawa opened the pull request:
# WHAT
Chat-space開発のスタートとなるindexページの作成
# WHY
サービスのメインとなるページのため、優先的に開発することが重要になるため。

KassemberKiyoshi committed:
[WIP]chat-space画面作成

# WHAT
   Chat-space開発のスタートとなるindexページの作成
# WHY
　サービスのメインとなるページのため、優先的に開発を行う必要があるため。

KassemberKiyoshi committed:
Chat-space 4/18時点最新

# WHAT
Chat-space開発のスタートとなるindexページの作成
# WHY
サービスのメインとなるページのため、優先的に開発することが重要になるため。

KassemberKiyoshi committed:
chat-space画面作成 4/18時点最新２

# WHAT
Chat-space開発のスタートとなるindexページの作成
# WHY
サービスのメインとなるページのため、優先的に開発することが重要になるため。

FUM01ichikawa commented:
前回分一部未修正（古いデータ）でしたので修正致しました。（div部分消す等）
https://github.com/FUM01ichikawa/chat-space/pull/7
それ以外は、下記の点修正しています。
https://github.com/FUM01ichikawa/chat-space/pull/4

・HTMLの最後はlink_toにできたらよかったのですが、上手く出来なかったのでアドバイスをいただけると非常に助かります。
・また、色については変数を使うことは出来たのですが、ファイルとして切り出し→importするのが上手く出来ていません。（ファイル作成＆アプリケーションファイルに記入はしたのですが、SCSS上から変数部分を消すと画面に反映されない※「定義されていない」というエラーが発生してしまいます・・

ken-nag commented on app/assets/stylesheets/_colors.scss:
```diff
@@ -0,0 +1,7 @@
+$sub_dark: #253141;
+$sub_light: #2f3e51;
+$main_text: #434A54;
+$sky_blue: #38AEF0;
```
改行を入れましょう

FUM01ichikawa replied:
改行をいれました。

ken-nag commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,41 @@
+        %label.image-label{for: :message_image}
+          %i.far.fa-images
+          %input#message_image.image-label__input{type: :file}
+        %a.message_box__send_button{:href =>"/index"}Send
```
改行を入れましょう

ken-nag commented on app/assets/stylesheets/application.scss:
```diff
@@ -0,0 +1,2 @@
+@import "./reset";
+@import "./style";
```
改行を入れましょう

ken-nag reviewed (commented):
お願いがあります！
新しくプルリクを出すときは以前のプルリクと切り分けないでプルリクを出して欲しいです。理由としては前回の変更を経て修正点を確認するのでreviewerとしては変更点が見にくくなってしまうからです

プルリクは一旦コードを書き終えたら出すもので、コメントのようにまだ出来てない時点で出してはいけないものです。厳しいようですが、実際の開発の現場でまだ修正が終わっていないものを出すことはなく、プルリク上でアドバイスすることは結構な工数になってしまいます

分からないことはコネクトで質問してみてください！

厳しい文章になってしまい大変申し訳ありません。分からないところを解決して修正されたプルリクお待ちしてます！

KassemberKiyoshi committed:
# WHAT
Chat-space開発のスタートとなるindexページの作成

# WHY
サービスのメインとなるページのため、優先的に開発することが重要になるため。

KassemberKiyoshi committed:
改行を追加

改行を追加

FUM01ichikawa commented:
改行を追加しましたが、エラーが起こっていた原因は、stylesheetフォルダの中に、同じ名前のファイルが２つ存在していたことでした。（.scssの前に、.cssが読まれていた？）
・colorsは無事インポートされ、画面は見える状態に正常化しました。
（質問したlink_toの部分はマストの内容ではないので大丈夫でした。）

shingoteshima commented on app/assets/stylesheets/_style.scss:
```diff
@@ -0,0 +1,199 @@
+  padding: 0 20px 0;
+}
+
+.subbox1__user_name{
```
&を使って書きましょう。他の箇所も同様です。

FUM01ichikawa replied:
＆を使って修正致しました。

shingoteshima commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,43 @@
+      .mainbox1__group_current
+        %h2.mainbox1__group_current_name 市川
+        %h3.mainbox1__group_current_members Member:ichi
+      %a.mainbox1__edit_button{:href =>"/index"} Edit
```
ハッシュロケット直しましょう。他の箇所も同様です。

FUM01ichikawa replied:
ハッシュロケットを修正致しました。
・25行目
・43行目

KassemberKiyoshi committed:
・ハッシュロケット変更 ＋”＆”記法活用

・hamlファイル→ハッシュロケット を変更
・SCSSファイル→アンパサンドの記法を使って修正

TOMARUMARU commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,43 @@
+          %i.far.fa-images        
+          %input#message_image.image-label__input{type: :file}
+          
+        %a.message_box__send_button{ href: "/index"}Send
```
link_toで書きましょう。他の箇所も同様です

KassemberKiyoshi committed:
aタグ部分→input＋type：submitで対応しました。

ご指摘いただいた下記点ですが、
・aタグ→link_toへの変更
上手く出来なかったため、input＋type:submitで対応致しました。
・メンターの方にも質問し、アドバイスもいただきましたが上手くいかず、別の方法を検討した次第です。

shingoteshima commented on app/assets/stylesheets/_style.scss:
```diff
@@ -0,0 +1,209 @@
+}
+
+.subbox1 {
+&__user_name{
```
上の `.subbox1` の中にネストしましょう。
他の箇所も修正しましょう。

FUM01ichikawa replied:
ネストが不足していが諸々の箇所について修正を加えました！

KassemberKiyoshi committed:
諸々ネストしました。

諸々ネストしました。（やっと意味がわかってきました。）
お手数をおかけしますが、ご確認お願い致します。

TOMARUMARU reviewed (approved):
LGTM
